pkg/virt-handler/migration-proxy/migration-proxy.go

```diff
@@ -29,7 +29,10 @@ import (
 	"strings"
 	"sync"
 
+	netutils "k8s.io/utils/net"
+
 	"kubevirt.io/client-go/log"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/network"
```

Comment:
It can be nice to have `network.IsIpv6Enabled()` in another place.
This import connect two completely different binaries

Reply:
This `IsIpv6Enabled` is so small that you can easily reimplement it here (or in a common library perhaps pkg/utils) and you already have `netutils "k8s.io/utils/net"` imported here.

Reply:
Hi, Thanks for your review,
so i took the time to check deeper and here are the results:
1. indeed pkg/virt-launcher/virtwrap/manager.go doesnt have it , i was mistaken because
pkg/virt-launcher/virtwrap/network/podinterface.go does have the var (not part of this PR),
(from what i was explained ,since Ihar change
makes the virt-handler runs code of virt-launcher in its network namespace, this is how it works, right ?)

2. pkg/virt-handler/migration-proxy/migration-proxy.go does have the VAR
3. eliminating both of this places which actually mux between [::] and 0.0.0.0 , the migration would still work,
so action items

1. i will remove the wrong place (1 above)
2. should i remove also the place which is right but isnt needed ? 
is there a case where the kernel will enforce ipv6 only and then 0.0.0.0 wont be usable ?

thanks

Reply:
> This `IsIpv6Enabled` is so small that you can easily reimplement it here (or in a common library perhaps pkg/utils) and you already have `netutils "k8s.io/utils/net"` imported here.

virt-handler has two places using 'IsIpv6Enabled' -
pkg/virt-launcher/virtwrap/network/podinterface.go &
pkg/virt-handler/migration-proxy/migration-proxy.go

IMO, we should reuse the same method in both of the places and not re-implement exactly the same code twice.
Since currently (unfortunately), the code under virt-launcher directory is a mixture of a code that is running on virt-launcher and code that is running on virt-handler I'm not sure the import is connecting completely different binaries.

@SchSeba @vladikr in what common place do you suggest to put it? Since the code is relevant to virt-handler only, I'm not sure pkg/utils is the best place.

Reply:
please consider the following when deciding:
1. even the addon at migration proxy, isnt mandatory , i mean that as i wrote before,
removing it , wont cause migration to fail under current circumstances.

2. having it in a common place would be prune to errors, of usage of it in places where the MY_POD_IP doesnt exists
since its a one liner i would suggest to leave it reimplemented or remove it totally since without it, migration works as well

Reply:
btw both virt-launcher and virt-handler pods have
KUBERNETES_SERVICE_HOST
according 
https://github.com/kubernetes/kubernetes/issues/40973
they are mandatory
their value is an IPv6 address of the cluster service
can we then just use in all places this env var in order to determine the address to be used ?
i mean both in podInterface (which was before me)
and also for virt-launcher to determine between 127.0.0.1 and ::1 
(in order to support cases where just one of them would be in the /etc/hosts and not both)
and for virt-handler to support cases where bindv6only will be true ?